sensor_msgs/Image Image
Mask[] masks